EventFilter/CTPPSRawToDigi/interface/RawDataUnpacker.h

```diff
@@ -26,7 +27,8 @@ class RawDataUnpacker
     typedef uint64_t word;
 
     /// VFAT transmission modes
-    enum { vmCluster = 0x80, vmRaw = 0x90 };
+    enum { vmCluster = 0x80, vmRaw = 0x90, vmDiamondCompact = 0xB0 };
+    enum { VFAT_DIAMOND_HEADER_OF_WORD_2=0x7800, VFAT_DIAMOND_HEADER_OF_WORD_3=0x7000, VFAT_DIAMOND_HEADER_OF_WORD_5=0x6800, VFAT_DIAMOND_HEADER_OF_WORD_7=0x6000, VFAT_HEADER_OF_EC=0xC000 };
```

Comment:
please add some linebreaks to improve readability.
Also, these are becoming long enough that it seems more appropriate to make these constexpr with explicit type specification.